tests/zinc/ErrorReporting.scala

```diff
@@ -0,0 +1,5 @@
+object Main {
+  private def printZooAnimals(animals: List[String]): Unit = println(s"The zoo has ${animals.mkString(",")}")
```

Comment:
Can we add a Scala 3 test for error reporting to make sure the nice Scala 3 error reporting works well still?

Reply:
Sure thing!